diff --git a/src/helpers/consts.h b/src/helpers/consts.h
index 1c5b19d..cf2eae2 100644
--- a/src/helpers/consts.h
+++ b/src/helpers/consts.h
@@ -2,9 +2,8 @@
 
 #define NAME "kabmat"
 #define VERSION "2.8.0"
-#define DATA_FILE (string(getenv("HOME")) + "/.local/share/kabmat/data")
-#define DATA_BACKUP_FILE                                                       \
-  (string(getenv("HOME")) + "/.local/share/kabmat/data_bkp")
+#define DATA_FILE "/usr/local/share/kabmat/data"
+#define DATA_BACKUP_FILE "/usr/local/share/kabmat/data_bkp"
 
 #define COLOR_PAIR_FOOTER 1
 #define COLOR_PAIR_MODE 2
